src/MongoDB/Manager.c

```diff
@@ -462,6 +474,21 @@ static void php_phongo_manager_free_object(void *object TSRMLS_DC) /* {{{ */
 	efree(intern);
 } /* }}} */
 
+#if PHP_VERSION_ID >= 70000
+zend_object* php_phongo_manager_create_object(zend_class_entry *class_type TSRMLS_DC) /* {{{ */
+{
+        php_phongo_manager_t *intern;
+
+        intern = (php_phongo_manager_t *)ecalloc(1, sizeof(php_phongo_manager_t)+zend_object_properties_size(class_type));
+
+        zend_object_std_init(&intern->std, class_type TSRMLS_CC);
+        object_properties_init(&intern->std, class_type);
+
+        intern->std.handlers = &php_phongo_handler_manager;
+
```

Comment:
Wait. Something is off here. Where is the free_object_storage defined?
We need to call php_phongo_manager_free_object() to cleanup ourself, mongoc things in particular